packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -32,6 +43,17 @@ export const onClientEntry = (
       });
     });
   }
+  // Inject the cookie consent scripts only if the page is served on the domain `*.commercetools.com`.
+  if (window && window.location.host.includes('.commercetools.com')) {
+    injectScript(
+      'https://cdn.cookielaw.org/consent/b104027d-4d10-4b75-9675-9ffef11562a8/OtAutoBlock.js'
+    );
+    injectScript(
+      'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js',
+      { 'data-domain-script': 'b104027d-4d10-4b75-9675-9ffef11562a8' },
+      'function OptanonWrapper() {};'
+    );
+  }
```

Comment:
This seems to do the trick.

![2020-04-08 13 19 21](https://user-images.githubusercontent.com/1110551/78778713-01c88900-799c-11ea-971a-b41f272bf165.gif)

I'd like to try this approach first.